src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,6 +35,7 @@ public RobotContainer() {
     //init output object
     m_output = new output();
     // Configure the button bindings
+    m_driveTrain = new driveTrain();
```

Comment:
Move this line up so that it is not under the configure button bindings comment.